stix_shifter_modules/qradar/stix_translation/json/events_from_stix_map.json

```diff
@@ -51,7 +51,7 @@
   },
   "artifact": {
     "fields": {
-      "payload_bin": ["utf8_payload"]
+      "payload_bin": ["eventpayload"]
```

Comment:
You'll need to add `flowsourcepayload` and `flowdestinationpayload` to the `flows_from_stix_map.json`. I see that originally we had the mapping using the UTF8 representation, but the STIX `payload_bin` property would normally be base64. If someone is using a pattern to search, wouldn't it use base64? We originally had a custom property for the utf8 payload so that it could be searched.

Reply:
I think I understand what you are suggesting. so we have implemented this as special case for qradar. though `payload_bin` supposed to be base64 but we expect people to perform text search in this pattern that's why we have utf8_payload mapped to artifact:payload_bin. `eventpayload` still doing the same. 
We can add a custom mapping `x-ibm-ariel:eventpayload` but that will work the same way

Reply:
And also will add this mapping for flows

Reply:
Yeah, so for the from-STIX map, `x-ibm-ariel:eventpayload` will map to the UTF8 (human readable) version of the payload and `artifact:payload_bin` will map to the base64 version.

Reply:
Same for flow source and destination payloads.